querydsl-sql/src/main/java/com/querydsl/sql/AbstractSQLQueryFactory.java

```diff
@@ -90,4 +91,61 @@ public final Connection getConnection() {
         return connection.get();
     }
 
+    /**
+     * Create a new SQLQuery instance with the given projection
```

Comment:
It's not returning a `SQLQuery` instance, it's returning a concrete impl of `AbstractSQLQuery` instance. Comment applies throughout changes to class.